java/serve/src/main/java/io/ray/serve/api/ServeControllerClient.java

```diff
@@ -60,16 +69,21 @@ public ServeControllerClient(BaseActorHandle controller, String controllerName)
                 .get();
   }
 
+  public DeploymentHandle getHandle(String deploymentName) {
+    return getHandle(deploymentName, Constants.SERVE_DEFAULT_APP_NAME, false);
+  }
+
   /**
-   * Retrieve RayServeHandle for service deployment to invoke it from Java.
+   * Retrieve DeploymentHandle for service deployment to invoke it from Java.
    *
    * @param deploymentName A registered service deployment.
+   * @param appName application name
    * @param missingOk If true, then Serve won't check the deployment is registered.
    * @return
    */
   @SuppressWarnings("unchecked")
-  public RayServeHandle getHandle(String deploymentName, boolean missingOk) {
-    String cacheKey = deploymentName + "#" + missingOk;
+  public DeploymentHandle getHandle(String deploymentName, String appName, boolean missingOk) {
+    String cacheKey = StringUtils.join(new Object[] {deploymentName, appName, missingOk}, "#");
```

Comment:
make "#" as a constant. There are lots of places using it, and replace them all.

Reply:
The "#" has been defined as a constant and replaced accordingly.